swagger/proto-config-gen.json

```diff
@@ -0,0 +1,26 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Umee SDK - gRPC Gateway docs",
+    "description": "A REST interface for state queries",
+    "version": "1.0.0"
+  },
+  "apis": [
+    {
+      "url": "./tmp-swagger-gen/umee/leverage/v1/query.swagger.json",
```

Comment:
why `tmp` prefix? It looks weird.

Reply:
it is just the name of the temp dir `/tmp-swagger-gen` used in `contrib/scripts/protoc-swagger-gen.sh` to generate the `./swagger/swagger.yaml`